components/widgets/resources/src/main/resources/resources/uicomponents/widgets/geneValidation.js

```diff
@@ -72,21 +82,6 @@ var XWiki = (function(XWiki) {
 
   (XWiki.domIsLoaded && init()) || document.observe("xwiki:dom:loaded", init);
   document.observe('xwiki:dom:updated', init);
-  document.observe('ms:suggest:selected', function(event) {
```

Comment:
Wouldn't a simple `Validate.Presence` be enough for variants? It seems that most of the code from `GeneVariantValidator` isn't needed, and mixing variants in there adds unneeded complexity.